src/main/kotlin/com/openlattice/mechanic/upgrades/DataExpirationUpgrade.kt

```diff
@@ -0,0 +1,23 @@
+package com.openlattice.mechanic.upgrades
+
+import com.openlattice.mechanic.Toolbox
+
+class DataExpirationUpgrade(private val toolbox: Toolbox) : Upgrade {
+    override fun upgrade(): Boolean {
+        toolbox.hds.connection.use{
+            val stmt = it.prepareStatement(addExpirationColumnsQuery())
+        }
+        return true;
+    }
+
+    override fun getSupportedVersion(): Long {
+        return Version.V2019_07_01.value
+    }
+
+    private fun addExpirationColumnsQuery(): String {
+        return "ALTER TABLE entity_sets " +
```

Comment:
This is fine (and not super important since this migration will be run once and then dropped), but it's better to use the values from PostgresTable and PostgresColumn to avoid typos / handle cases where one of those values is updated (i.e. "ALTER TABLE ${PostgresTable.ENTITY_SETS.name} ADD COLUMN ${PostgresColumn.blah.whatever} etc"

Reply:
Oh yes. I will change that just to build better habits